x-pack/plugins/index_lifecycle_management/server/routes/api/index/register_retry_route.ts

```diff
@@ -29,24 +28,21 @@ const bodySchema = schema.object({
   indexNames: schema.arrayOf(schema.string()),
 });
 
-export function registerRetryRoute({ router, license, lib }: RouteDependencies) {
+export function registerRetryRoute({ router, license }: RouteDependencies) {
   router.post(
     { path: addBasePath('/index/retry'), validate: { body: bodySchema } },
     license.guardApiRoute(async (context, request, response) => {
       const body = request.body as typeof bodySchema.type;
       const { indexNames } = body;
 
       try {
-        await retryLifecycle(
-          context.core.elasticsearch.legacy.client.callAsCurrentUser,
-          indexNames
-        );
+        await retryLifecycle(context.core.elasticsearch.client.asCurrentUser, indexNames);
         return response.ok();
       } catch (e) {
-        if (lib.isEsError(e)) {
+        if (e instanceof ResponseError) {
```

Comment:
I think there are other instances of `lib.isEsError` being used throughout the server code - do they need to be updated as well?